src/main/java/frc/robot/commands/IntakeSequence.java

```diff
@@ -1,46 +1,54 @@
 package frc.robot.commands;
 
-import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Cal;
 import frc.robot.subsystems.Intake;
 import frc.robot.subsystems.Lift;
+import frc.robot.subsystems.Lift.LiftPosition;
 
-public class IntakeSequence extends CommandBase {
+public class IntakeSequence extends SequentialCommandGroup {
   private Intake intake;
   private Lift lift;
 
-  public IntakeSequence(Intake intake, Lift lift) {
-    this.intake = intake;
-    this.lift = lift;
-  }
+  // deploy intake for specified amount of time
+  private ParallelRaceGroup deployIntake =
+      new RunCommand(intake::deploy, intake).withTimeout(Cal.Intake.AUTO_CLAMP_WAIT_TIME_SECONDS);
 
-  @Override
-  public void initialize() {
-    intake.deploy();
-    lift.setDesiredPosition(Lift.LiftPosition.GRAB_FROM_INTAKE);
-  }
+  // run intake infinitely (until parallel command ends)
+  private RunCommand runIntake = new RunCommand(intake::intakeGamePiece, intake);
```

Comment:
Since intakeGamePiece just sets the power of the intake, we think this can just be an instant command because it doesn't need to be called repeatedly.